CONTRIBUTING.md

```diff
@@ -1,4 +1,4 @@
-# Contributing to GIT Simplified (GITS)
+# Contributing to Ideal ReposItory for Software projects (I.R.I.S)
```

Comment:
Rename it to GITS2.0 - Ideal ReposItory for Software projects (I.R.I.S) or something similar

Reply:
Good catch. Made the changes, thanks!